### PR TITLE
[ci skip] Doc loki

### PR DESCRIPTION
**Pull request by nadegepepin**
First version

### CODE REVIEW

File: doc/docs/master/deploy-manage/deploy/loki.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
+!!! note To deploy and configure a Pachyderm cluster
+to forward logs to Loki,
```
Comment (echohack) [inline doc/docs/master/deploy-manage/deploy/loki.md:2]: forward -> ship

File: doc/docs/master/deploy-manage/deploy/loki.md [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+
+## Fetching logs
+
+While enabling Loki will enable the collection of logs, commands such as `pachctl logs` will not fetch logs directly from Loki until the `LOKI_SERVICE_HOST` and `LOKI_SERVICE_PORT` environment variables are set on the `pachd` container.
+
+For example, a `deployment.json` generated with 
+```shell
+    pachctl deploy local --dry-run > deployment.json`     
+```
+can be modified to include log forwarding as follows:
```
Comment (echohack) [inline doc/docs/master/deploy-manage/deploy/loki.md:33]: can be modified to include log forwarding as follows: -> can be modified to make logs available for Loki as follows:

**Review by echohack — CHANGES_REQUESTED** [review]
Just a few fixes around wording of forwarding. Sorry for not fixing this sooner!

**Review by echohack — APPROVED** [review]
👍

**Comment by nadegepepin** [comment]
Closing this PR - was merged as part of the 1.12.0 release PR #5534

**Pull request closed without merging**